model/tasks/ImportAndCompile.php

```diff
@@ -96,9 +97,9 @@ public function __invoke($params)
                 );
             }
 
-            $label = 'Delivery of ' . $test->getLabel();
-            $parent = $this->checkSubClasses($params[self::OPTION_DELIVERY_LABELS]);
-            $deliveryFactory = $this->getServiceManager()->get(DeliveryFactory::SERVICE_ID);
+            $label             = 'Delivery of ' . $test->getLabel();
+            $parent            = $this->checkSubClasses($params[self::OPTION_DELIVERY_LABELS]);
+            $deliveryFactory   = $this->getServiceManager()->get(DeliveryFactory::SERVICE_ID);
```

Comment:
I'm factually a bit cross-eyed like Sid from "Ice Age", but still, my brain prefers the version without random length whitespace :) A matter of habit, but I think I'm with majority here (could be wrong)